### PR TITLE
Radstorm fix

### LINKED ISSUE #2778: Maintenance access after radstorms
This may just be awkwardly hard to solve, but after a radiation storm hits the station, when it normally enables the emergency maintenance access, it revokes it. This is fine, but it revokes it regardless of whether the time it was enabled was due to the radstorm trigger or not.

### PR DESCRIPTION
**Pull request by CrAzYPiLoT-SS13**
Fixes #2778

### CODE REVIEW

File: code/modules/events/radiation_storm.dm [hunk 1]
```diff
@@ -67,4 +69,8 @@
 
 		sleep(600) // Want to give them time to get out of maintenance.
 
-		revoke_maint_all_access()
+		if(is_radstorm)
+			revoke_maint_all_access()
+			is_radstorm = 0
+		else
```
Comment (tigercat2000) [inline code/modules/events/radiation_storm.dm]: This `else return` block is completely pointless and stops the function from exiting normally

Comment (CrAzYPiLoT-SS13) [inline code/modules/events/radiation_storm.dm]: Yeah, now that I think of it, I don't know why I put it there.

**Comment by CrAzYPiLoT-SS13** [comment]
This is merge-ready, just so you know.

**Comment by Fox-McCloud** [comment]
Adding in a global var for such a minor issue; no dice :-1:

**Comment by CrAzYPiLoT-SS13** [comment]
Well, I don't know any other way to coordinate two files, excuse my ignorance. Mind explaining any other way? BTW, why is a global var such a problem?

**Comment by CrAzYPiLoT-SS13** [comment]
Bump.

**Comment by tigercat2000** [comment]
There isn't a cleaner solution currently available, the maint access system would need to get an overhead controller registering why they are locked/unlocked/whatever. However, for such a minor issue, that much overhead or this really dirty solution is really not worth it.

**Pull request closed without merging**